jacodb-taint-configuration/src/main/kotlin/org/jacodb/taint/configuration/Position.kt

```diff
@@ -23,31 +23,55 @@ fun interface PositionResolver<out R> {
     fun resolve(position: Position): R
 }
 
+/**
+ * A class representing a position of tainted data in a method.
```

Comment:
This is not a class

Maybe simply "A representation of a position of tainted data"?